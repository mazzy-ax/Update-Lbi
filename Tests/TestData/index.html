<html>
<body>
    <p>some text content with <!-- #BeginLibraryItem "/Library/onliner.lbi" -->is it updated from lbi-file?<!-- #EndLibraryItem -->.</p>
    anoter text<!-- #BeginLibraryItem "/Library/testdiv.lbi" -->is it indented?<!-- #EndLibraryItem -->
</body>
</html>
