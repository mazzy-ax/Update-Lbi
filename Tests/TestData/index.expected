<html>
<body>
    <p>some text content with <!-- #BeginLibraryItem "/Library/onliner.lbi" --><span>onliner!</span><!-- #EndLibraryItem -->.</p>
    another text<!-- #BeginLibraryItem "/Library/testdiv.lbi" -->
    <div>
        div line 1
        div line 2
    </div>
    <!-- #EndLibraryItem -->
</body>
</html>
